apps/storybook/.storybook/main.ts

```diff
@@ -17,7 +17,7 @@ const storiesGlob = 'src/**/*.stories.@(js|jsx|ts|tsx)'
 const config: StorybookConfig = {
   framework,
   stories: packagesWithStories.map((packageFolder) => `../../../packages/${packageFolder}/${storiesGlob}`),
-  addons: storykitAddons,
+  addons: [...storykitAddons, 'storybook-addon-pseudo-states'],
```

Comment:
Any reason not to add this to `storykitAddons`?

Reply:
i agree with this sentiment. if there are challenges, we could defer, but it should be our preferred approach as we'll need it in pcln-web